### PR TITLE
[DH-956] Rename address fields in Investment Project.

### PR DESCRIPTION
**Pull request by elcct**
This renames address fields in Investment Project to make it consistent with address fields in other entities.

### CODE REVIEW

File: datahub/investment/migrations/0020_auto_20171018_1506.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            new_name='address_1',
+        ),
+        migrations.RenameField(
+            model_name='investmentproject',
+            old_name='address_line_2',
+            new_name='address_2',
+        ),
+        migrations.RenameField(
+            model_name='investmentproject',
+            old_name='address_line_3',
```
Comment (reupen) [inline datahub/investment/migrations/0020_auto_20171018_1506.py]: Looks like the migration has swapped address_3/postcode in the new column names.

File: fixtures/test_data.yaml [hunk 2]
```diff
@@ -561,9 +561,9 @@
     project_manager: d7493b4e-5d7b-4834-98d9-28b78a74052a
     project_assurance_adviser: 5a644146-5298-4741-91f3-d5d7558adf47
     uk_company: a73efeba-8499-11e6-ae22-56b6b6499611
-    address_line_1: 10 Eastings Road
-    address_line_2: London
-    address_line_postcode: W1 2AA
+    address_1: 10 Eastings Road
+    address_2: London
```
Comment (reupen) [inline fixtures/test_data.yaml]: Should probably be address_town here.

File: datahub/investment/migrations/0020_auto_20171018_1506.py (hunk 1 above)
Comment (elcct) [reply datahub/investment/migrations/0020_auto_20171018_1506.py]: Wow, good catch! I'll update in a minute. Thanks.

File: datahub/investment/validate.py [hunk 3]
```diff
@@ -35,9 +35,9 @@
     'non_fdi_r_and_d_budget': Stage.verify_win.value,
     'new_tech_to_uk': Stage.verify_win.value,
     'export_revenue': Stage.verify_win.value,
-    'address_line_1': Stage.verify_win.value,
-    'address_line_2': Stage.verify_win.value,
-    'address_line_postcode': Stage.verify_win.value,
+    'address_1': Stage.verify_win.value,
+    'address_2': Stage.verify_win.value,
```
Comment (reupen) [inline datahub/investment/validate.py]: Should probably be address_town here instead of address_2 now.

Comment (ztolley) [reply datahub/investment/validate.py]: I think we need address_1, address_2, address_town and address_postcode

Comment (reupen) [reply datahub/investment/validate.py]: We don't require address_2 anywhere else. There won't always be a second line.

File: fixtures/test_data.yaml (hunk 2 above)
Comment (elcct) [reply fixtures/test_data.yaml]: True, it looks like it should be `address_town` instead.

File: datahub/investment/validate.py (hunk 3 above)
Comment (elcct) [reply datahub/investment/validate.py]: I changed it to `address_town` as well. `address_2` is now not required.

**Comment by ztolley** [comment]
You need address_1 AND address_2 for some addresses IMHO, plus all the other address entries have 1 and 2. I know for my home address you need two

**Comment by elcct** [comment]
@ztolley `address_2` do exist, but it is not required.

**Pull request merged**